### PR TITLE
Fix parameter_input dropped in new workflow form

### PR DESCRIPTION
**Pull request by mvdbeek**
Fixes https://github.com/galaxyproject/galaxy/pull/9151#issuecomment-677635591

### CODE REVIEW

**Comment by jmchilton** [comment]
Sorry about this - thanks for the fix.

File: client/src/components/Workflow/Run/WorkflowRunFormSimple.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <script>
 import Form from "mvc/form/form-view";
 import { invokeWorkflow } from "./services";
+import isWorkflowInput from "components/Workflow/constants";
```
Comment (dannon) [inline client/src/components/Workflow/Run/WorkflowRunFormSimple.vue]:
```suggestion
import { isWorkflowInput } from "components/Workflow/constants";
```

File: client/src/components/Workflow/constants.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const WorkflowInputs = ["data_input", "data_collection_input", "parameter_input"];
+
+function isWorkflowInput(stepType) {
+    return WorkflowInputs.includes(stepType);
+}
+
+export { isWorkflowInput as default };
```
Comment (dannon) [inline client/src/components/Workflow/constants.js]:
```suggestion
const WorkflowInputs = ["data_input", "data_collection_input", "parameter_input"];

export function isWorkflowInput(stepType) {
    return WorkflowInputs.includes(stepType);
}
```

File: client/src/components/Workflow/Run/WorkflowRunFormSimple.vue (hunk 1 above)
Comment (dannon) [reply client/src/components/Workflow/Run/WorkflowRunFormSimple.vue]: Not sure I grok `constants` either -- what else goes in here?  Might be more `util`?

Comment (mvdbeek) [reply client/src/components/Workflow/Run/WorkflowRunFormSimple.vue]: Yeah, I initially just had the array in there, but thought a function might be nicer. I can make this util, or, considering we're not re-suing this yet anywhere, I could also add this straight to WorkflowRunFormSimple.vue. Seems like we'd need this for invocation step displaying as well, but I can move this when we actually need it.

**Pull request merged**